recipes/oslo.config/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{%set name = "oslo.config" %}
+{%set version = "3.17.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "a45cb90cf999536f0f7dd3d3e578d4645d2228474f9fafba982e73c5aa1fef7a" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - debtcollector >=1.2.0
+    - netaddr !=0.7.16,>=0.7.12
+    - six >=1.9.0
+    - stevedore >=1.10.0
+    - oslo.i18n >=2.1.0
+    - rfc3986 >=0.2.2
+
+test:
+  imports:
+    - oslo_config
+    - oslo_config.cfg
+    - oslo_config.types
+
+about:
+  home: http://docs.openstack.org/developer/oslo.config/
+  license: Apache 2.0
+  # license_file: No MANIFEST.in
```

Comment:
It is! You aren't insane - it just doesn't seem to be in the github mirror, I guess, and since I was checking github...